source/educators/how-tos/course_development/configure_discussion_legacy.rst

```diff
@@ -1,10 +1,12 @@
-.. _Working with Discussion Components:
+.. _Configure Open edX Discussions Legacy:
 
-Working with Discussion Components
-###################################
+Configure Open edX Discussions - Legacy
+########################################
 
 .. tags:: educator, how-to
 
+.. note:: This is the legacy way to create a discussion component. View the latest way to create a discussion component in :ref:`Configure Open edX Discussions`.
```

Comment:
I think "Legacy" is potentially confusing/not specific. It would be beter to say "This is the way to create discussion components on older versions of the Open edX platform, pre-(release name) release". I don't remember what release the new one debuted in

Reply:
Quince: https://docs.openedx.org/en/latest/community/release_notes/quince.html#learner-experiences

Reply:
I fixed this in a previous PR